pay-api/src/pay_api/models/routing_slip.py

```diff
@@ -52,6 +52,10 @@ class RoutingSlip(Audit):  # pylint: disable=too-many-instance-attributes
     parent_number = db.Column(db.String(), ForeignKey('routing_slips.number'), nullable=True)
     refund_amount = db.Column(db.Numeric(), nullable=True, default=0)
     total_usd = db.Column(db.Numeric(), nullable=True)  # Capture total usd payments if one of payments has USD payment
+    # It's not possible to update a receipt's amount or number in CAS (PUT/PATCH).
+    # Allows to create a new receipt in CAS for the same routing slip number.
+    # Earlier versions should be adjusted to zero before increasing the cas_version_suffix.
+    cas_version_suffix = db.Column(db.Integer, default=1)
```

Comment:
This field we'll probably use for corrections. We'll have to increment it and add it to the end of the routing slips if it's > 1. 
Receipt Numbers or routing slip numbers are unique across CFS accounts.